### PR TITLE
Update v3.8.0 (unreleased) changelog for style

### PR DESCRIPTION
**Pull request by m-green**
This updates the v3.8.0 (unreleased) changelog for style and readability.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,38 +4,53 @@
...
+You can now turn spellcheck on or off in the input, textarea and character count components using the new `spellcheck` option instead of the `attributes` option.
+
+For example:
+
+```javascript
+{{ govukInput
+  {{
+    "spellcheck": true
+  }}
+}}
```
Comment (36degrees) [inline CHANGELOG.md:37]: The options passed to `govukInput` should be wrapped in a single curly braces and brackets, like `govukInput({ spellcheck: true })`.

There's no need to quote spellcheck.

To be consistent with the examples in the Design System, we'd format it like this:

```suggestion
{{ govukInput({
    "spellcheck": true
}) }}
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,38 +4,53 @@
...
+- [#1859: Add spellcheck option](https://github.com/alphagov/govuk-frontend/pull/1859)
+- [#1869: Add missing spellcheck param to character count](https://github.com/alphagov/govuk-frontend/pull/1869)
 
 ### Deprecated features
 
-#### The `$govuk-border-width-form-element-error` setting
+#### $govuk-border-width-form-element-error
 
-The `$govuk-border-width-form-element-error` setting has been deprecated, and will be removed in GOV.UK Frontend v4.0.
+From GOV.UK Frontend v4.0.0, you'll no longer be able to use the [`$govuk-border-width-form-element-error`](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-border-width-form-element-error) Sass setting to set the border of a form group with an error.
```
Comment (36degrees) [inline CHANGELOG.md:49]: I think it's more likely that users might be _referencing_ the setting in their own components than they've actual set it to anything other than the default.

The following paragraph also relates to updating places in their own code where they're referencing it – if they are currently _setting_ `$govuk-border-width-form-element-error` in their code then replacing that probably isn't what we want them to do.

File: CHANGELOG.md (hunk 1 above)
Comment (m-green) [reply CHANGELOG.md:37]: Sorry I didn't check this carefully enough! Thanks for spotting - I'll fix it up.

File: CHANGELOG.md (hunk 2 above)
Comment (m-green) [reply CHANGELOG.md:49]: Thank you - will update!

Comment (m-green) [reply CHANGELOG.md]: Updated in 41e92e8

File: CHANGELOG.md (hunk 1 above)
Comment (m-green) [reply CHANGELOG.md]: Updated in 41e92e8

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,38 +4,51 @@
...
 ### New features
 
-#### Darkened secondary text colour
+#### The secondary text colour is now darker
 
-We darkened `govuk-colour("dark-grey")` to improve the readability of hint text. It now has a contrast ratio of 7:1 and helps hint text meet the WCAG 2.1 (AAA) accessibility standard.
+`$govuk-secondary-text-colour` and `govuk-colour("dark-grey")` are now darker so users can more clearly read hint text that uses the colour.
 
-#### Updated error styling for text input, select, textarea and file upload components
+The colour now has a contrast ratio of 7:1, and helps hint text meet the WCAG 2.1 (AAA) accessibility standard.
```
[the remark this answers is not in the document]
Comment (m-green) [reply CHANGELOG.md]: Good plan - added!

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,38 +4,51 @@
 
 ### New features
 
-#### Darkened secondary text colour
+#### The secondary text colour is now darker
```
Comment (vanitabarrett) [inline CHANGELOG.md:7]: Do we need to link to the pull request where this was changed? (we seem to do this for the other entries)

Comment (m-green) [reply CHANGELOG.md:7]: Thanks Vanita - added in 9892ac3

**Comment by m-green** [comment]
@36degrees @vanitabarrett This has been 2i'd now! Ok to merge?

**Pull request merged**